### PR TITLE
FIM System tests: Scenario 0206 Tag option

### LINKED ISSUE #524: FIM System tests. 0206: Tag option
Hello team,

This issue is related to #493. It will contain the advances related to the creation of the scenario 206.

### Objectives

Implement the scenarion 206.

###  Tasks

- [x] Create ossec.conf
- [x] Create `launch_scenario.yml`
- - [x] Create/Delete/modify the necessary files
- - [x] Check alerts in `alerts.json`.
- - [x] Check alerts in `Elasticsearch`.

Regards,
Daniel Folch

### PR DESCRIPTION
**Pull request by DFolchA**
Hello team,

This PR closes #524, I have created the necessary files and directories to launch the scenario 206 which tested that the tags included in the syscheck configuration showed in the alerts.

Regards, 
Daniel Folch

### CODE REVIEW

File: tests/system/fim/common_tasks/verify_alerts.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@
   - name: "Verify alert json log | {{ event }}"
     script: >
       verify_alerts_json.py -i {{ item.path }} -e {{ event }} 
-      -o {{ missing_alerts_json_path }}
+      -o {{ missing_alerts_json_path }} {{ verify_json_script_extra_params }}
```
Comment (jm404) [inline tests/system/fim/common_tasks/verify_alerts.yml]: Let's standardize this as `json_verification_extra_arguments`

File: tests/system/fim/common_tasks/verify_alerts.yml [hunk 2]
```diff
@@ -21,7 +21,8 @@
     script: >
       verify_alerts_elasticsearch.py -i {{ item.path }} -e {{ event }} 
       -a {{ hostvars[groups["elasticsearch"][0]]["private_ip"] }}
-      -o {{ missing_alerts_elasticsearch_path }}
+      -o {{ missing_alerts_elasticsearch_path }} 
+      {{ verify_elastic_script_extra_params }}
```
Comment (jm404) [inline tests/system/fim/common_tasks/verify_alerts.yml]: Lets standardize this as `elastic_verification_extra_arguments`

File: tests/system/fim/common_tasks/verify_alerts.yml (hunk 1 above)
Comment (DFolchA) [reply tests/system/fim/common_tasks/verify_alerts.yml]: Done in https://github.com/wazuh/wazuh-qa/pull/552/commits/9455e81de250cfb7a9077da377925afe70ae6782.

File: tests/system/fim/common_tasks/verify_alerts.yml (hunk 2 above)
Comment (DFolchA) [reply tests/system/fim/common_tasks/verify_alerts.yml]: Done in https://github.com/wazuh/wazuh-qa/pull/552/commits/9455e81de250cfb7a9077da377925afe70ae6782.

**Pull request merged**